pkg/workload/schemachange/operation_generator.go

```diff
@@ -2671,6 +2671,24 @@ func makeOpStmt(queryType opStmtType) *opStmt {
 	}
 }
 
+// opStmtFromTree constructs an operation from the provide tree.Statement.
+func newOpStmt(stmt tree.Statement, codes codesWithConditions) *opStmt {
```

Comment:
nit: do you think it's better if we rename `codes` to `expectedExecErrCodes`?

Reply:
Done.